BundID/Views/Home/HomeView.swift

```diff
@@ -18,14 +17,16 @@ struct HomeState: Equatable {
 }
 
 extension HomeState: AnalyticsView {
-     var route: [String] {
+    var route: [String] {
         []
-     }
- }
+    }
+}
 
 enum HomeAction: Equatable {
     case triggerSetup
-    case triggerIdentification(tokenURL: String)
+#if PREVIEW
+    case triggerIdentification
```

Comment:
I would prefer to have the tokenURL in that action here and pass in the demoTokenURL in all cases.